lua/conform/init.lua

```diff
@@ -721,10 +721,17 @@ M.formatexpr = function(opts)
   end
   local end_line = vim.fn.getline(end_lnum)
   local end_col = end_line:len()
-  opts.range = {
-    start = { start_lnum, 0 },
-    ["end"] = { end_lnum, end_col },
-  }
+
+  if vim.v.count == vim.fn.line("$") then
```

Comment:
Shouldn't `vim.v.lnum` (`start_lnum`) also be checked?

Reply:
I don't think so because it's already checking whether the number of lines to be formatted (`vim.v.count`) is equal to the total number of lines (`vimv.fn.line("$")`). Thus it must be formatting the whole buffer, no?

Reply:
I see, you're right. The count is not the last line but the length of the range, so it's only when the range represents the whole lines.

Reply:
So do you think it's good? Solves my issue and doesn't seem to break anything for me.